further_link/endpoint/upload.py

```diff
@@ -1,9 +1,17 @@
 import json
+from os import path
 
 from aiohttp import web
 
 from ..util.upload import BadUpload, directory_is_valid, do_upload
-from ..util.user_config import get_working_directory
+from ..util.user_config import default_user, get_working_directory
+
+
+def is_miniscreen_project(directory):
+    for filename in directory.get("files", {}):
+        if filename.endswith(".cfg"):
+            return True
```

Comment:
actually it can be `*.cfg`; however I see the benefit in only looking for `project.cfg` files, we should probably change it to specifically look for that file (here & in the miniscreen app)